services/node/node.service.js

```diff
@@ -20,33 +22,66 @@ module.exports = class NodeVersion extends NPMBase {
     return [
       {
         title: 'node',
-        previewUrl: 'passport',
-        keywords: ['npm'],
+        pattern: ':packageName',
+        namedParams: { packageName: 'passport' },
+        staticExample: this.renderStaticExample({
+          nodeVersionRange: '>= 6.0.0',
+        }),
+        keywords,
       },
       {
         title: 'node (scoped)',
-        previewUrl: '@stdlib/stdlib',
-        keywords: ['npm'],
+        pattern: '@:scope/:packageName',
+        namedParams: { scope: 'stdlib', packageName: 'stdlib' },
+        staticExample: this.renderStaticExample({
+          nodeVersionRange: '>= 6.0.0',
+        }),
+        keywords,
       },
       {
         title: 'node (tag)',
-        previewUrl: 'passport/latest',
-        keywords: ['npm'],
+        pattern: ':packageName/:tag',
+        namedParams: { packageName: 'passport', tag: 'latest' },
+        staticExample: this.renderStaticExample({
+          nodeVersionRange: '>= 6.0.0',
+          tag: 'latest',
+        }),
+        keywords,
       },
       {
         title: 'node (scoped with tag)',
-        previewUrl: '@stdlib/stdlib/latest',
-        keywords: ['npm'],
+        pattern: '@:scope/:packageName/:tag',
+        namedParams: { scope: 'stdlib', packageName: 'stdlib', tag: 'latest' },
+        staticExample: this.renderStaticExample({
+          nodeVersionRange: '>= 6.0.0',
+          tag: 'latest',
+        }),
+        keywords,
       },
       {
         title: 'node (scoped with tag, custom registry)',
-        previewUrl: '@stdlib/stdlib/latest',
+        pattern: '@:scope/:packageName/:tag',
+        namedParams: { scope: 'stdlib', packageName: 'stdlib', tag: 'latest' },
         query: { registry_uri: 'https://registry.npmjs.com' },
-        keywords: ['npm'],
+        staticExample: this.renderStaticExample({
+          nodeVersionRange: '>= 6.0.0',
+          tag: 'latest',
+        }),
+        keywords,
       },
     ]
   }
 
+  static renderStaticExample({ tag, nodeVersionRange }) {
+    // This should match the behavior of `async render()`, which is enforced
+    // with a unit test.
```

Comment:
It took me a while to work out that the reason why we have to do this non-standard thing is because `render()` has to call an async function but we need this to be a synchronous call. Can we make this extra clear in the comment here.

Reply:
I updated the comment. See if that's clearer?